superweather/superweather/LocationTool.h

```diff
@@ -0,0 +1,21 @@
+//
+//  LocationTool.h
+//  superweather
+//
+//  Created by mikael petersson on 2016-04-17.
+//  Copyright © 2016 mikael petersson. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+@interface LocationTool : NSObject<CLLocationManagerDelegate>
+
+@property (strong, nonatomic) CLLocationManager *locationManager;
+
+
+-(LocationTool*)init;
```

Comment:
LocationTool inherits an init method as a public method from NSObject so you don't need to redeclare it incase you wont change the return type.